src/embedding_service.py

```diff
@@ -76,6 +76,28 @@ async def route_openai_get_embeddings(
                 embeddings, model=model_name, usage=usage
             )
 
+    async def route_get_image_embeddings(
+        self,
+        image_input: str | list[str],
+        model_name: str,
+        return_as_list: bool = False,
+    ):
+        """returns embeddings for the input image urls"""
+        if not self.is_running:
+            await self.start()
+        if not isinstance(image_input, list):
+            image_input = [image_input]
+
+        embeddings, usage = await self.engine_array[model_name].image_embed(image_input)
```

Comment:
This doesn't seem to work for me. Chainging the argument to images=image_input resolves the issue on my end.